cmd/support-callhome.go

```diff
@@ -91,20 +92,43 @@ type supportCallhomeMessage struct {
 
 // String colorized callhome command output message.
 func (s supportCallhomeMessage) String() string {
-	var msg string
 	if s.Action == "status" {
-		msgs := []string{}
+		columns := []table.Column{
+			{Title: "Features", Width: 15},
+			{Title: "", Width: 10},
+		}
+
+		rows := []table.Row{}
+
 		if len(s.Diag) > 0 {
-			msgs = append(msgs, "Diagnostics is "+s.Diag)
+			rows = append(rows, table.Row{licInfoField("Diagnostics"), licInfoVal(s.Diag)})
 		}
 		if len(s.Logs) > 0 {
-			msgs = append(msgs, "Logs is "+s.Logs)
+			rows = append(rows, table.Row{licInfoField("Logs"), licInfoVal(s.Logs)})
 		}
-		msg = strings.Join(msgs, "\n")
-	} else {
-		msg = s.Feature + " is now " + s.Action
+
+		t := table.New(
```

Comment:
We need to call `initLicInfoColors()` in the beginning; otherwise the two rows (diagnostics and logs) get printed in different colors. The side effect of this is that the table column widths will have to be a little bigger; otherwise the values in cells get truncated. `20, 15` instead of `15, 10` would work.

Reply:
![callhome-status](https://github.com/minio/mc/assets/355479/3e2cc632-80ba-49cd-a687-0de70b24650c)
screenshot of the rows in different colors

Reply:
updated @anjalshireesh